presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -278,6 +278,23 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testColumnComment()
```

Comment:
Can you please also add a test for SqlServer

Reply:
Confirmed SQLServer always return null on REMARKS column and this is related issue https://github.com/Microsoft/mssql-jdbc/issues/646. Perhaps, we need to retrieve column comment from sys.extended_properties tables.